ETC.tsv

```diff
@@ -3377,8 +3377,8 @@ ETC_20150317_003376	Wiping with scaffold
 ETC_20150317_003377	{memo X}Hook charcoal
 ETC_20150317_003378	It's not enough!
 ETC_20150317_003379	Amount charged is not enough!
-ETC_20150317_003380	Box Checking
-ETC_20150317_003381	Brother found the journal of brother!
+ETC_20150317_003380	$Checking the Box
```

Comment:
I like it this way. "Checking the Box" - I've done this myself too, for example: "Activating the Altar".